### DISCUSSION TITLE
Not launching.

### DISCUSSION THREAD

**Discussion by LaykaKiro** [SteamRE/DepotDownloader#288] [Q&A] [answered]
Like it says on the box. It just doesn't launch. I see a cmd window for what could easily be a split second then the process dies.

**Comment by psychonic** [comment, marked as answer]
What command arguments did you pass? If none, that is normal behavior.

**Reply by LaykaKiro** [reply]
Command arguments? May I request that you elaborate? I don't really have the chance to type anything in, it just opens then closes, quicker than I can really register...

**Reply by psychonic** [reply]
Usage instructions are in the README file, the contents of which are also displayed on the project page. If you're not familiar with command arguments, you may have a bad time.

**Reply by LaykaKiro** [reply]
Oh, that. Nono. I mean it just doesn't launch. I have literally no time to put in any command arguments...

**Reply by psychonic** [reply]
That's not how command arguments work. They're not prompted for. You have to specify them at the time of launch.

**Reply by LaykaKiro** [reply]
Wait, am I meant to open a cmd window in the folder and not launch the .exe?

**Reply by psychonic** [reply]
> Wait, am I meant to open a cmd window in the folder and not launch the .exe?

This.

**Reply by LaykaKiro** [reply]
Deadass, didn't think that was what I had to do. I read the Readme five different times.
